docs/src/man/comparisons.md

```diff
@@ -0,0 +1,27 @@
+The table compares the main functions of DataFrames with R dplyr and Stata
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean)`  | `summarize(mean(x))`    | `collapse (mean) x `|
+|`transform(df, :x => mean)`   | `mutate(mean(x))`    | `egen x_mean = mean(x)`|
+|`select(df, :x, :y)`   | `select(df, x, y)`  | `keep x y` |
+|`sort(df, :x)`   | `arrange(df, x)`    | `sort x`|
+|`filter(:x => x -> x >= 1, df)`   | `filter(df, x >= 1)`  | `keep if x >= 1` |
+
+Here are more complicated examples for `combine`/`transform`/`select`:
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean => :x_mean)`   | `summarize(df, x_mean = mean(x))`    | `collapse (mean) x_mean = x`|
+|`combine(df, :x => x -> maximum(x) - minimum(x))`   | `summarize(df, max(x) - min(x))`    | |
+|`combine(df, [:x, :y] => cov)`   | `summarize(df, cov(x, y))`    | |
+|`combine(df, :x => maximum,  :y => minimum)`   | `summarize(df, max(x), min(y))`    | `collapse (max) x (min) y` |
+|`combine(df, [:x, :y] .=> mean)`   | `summarize(df, across(c(x, y), mean))`    | `collapse (mean) x y` |
+|`combine(df, names(df, r"^x.*") .=> mean)`   | `summarize(df, across(starts_with("x"), mean))`    | `collapse (mean) x*` |
+
+As in dplyr and, the functions `select`, `transform`, `combine` can also be used on grouped dataframes:
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`transform(groupby(df, :id), :x => mean)`   | `mutate(group_by(df, id), mean(x))`    | `egen x_mean = mean(x), by(id)`|
+|`combine(groupby(df, :id), :x => mean)`  | `summarize(group_by(df, id), mean(x))`    | `collapse (mean) x, by(id)`|
```

Comment:
I would put the functions so that `combine` is first or last, as it does a different thing than `select` and `transform`.